pkg/wsman/hash.go

```diff
@@ -68,7 +68,8 @@ func (c *authChallenge) response(method, uri, cnonce string) (string, error) {
 		hashedCredentials := c.hashCredentials()
 		hashedURI := c.hashURI(method, uri)
 		nonceData := c.Nonce
-		if c.Qop == "auth" {
+		if strings.Contains(c.Qop, "auth") || strings.Contains(c.Qop, "auth-int") {
```

Comment:
same comment as above, auth only is enough

Reply:
removed auth-int check